modules/ocf_tv/manifests/pulse.pp

```diff
@@ -1,11 +1,40 @@
 class ocf_tv::pulse {
   include ocf::packages::pulse;
 
+  # pulseaudio as a systemd service fixes a number of
+  # problems with the daemon not starting properly
+  # or starting with the wrong environment
+  ocf::systemd::service { 'pulseaudio':
+    source => 'puppet:///modules/ocf_tv/pulseaudio.service',
+    require => [
+      Package['pulseaudio'],
+      User['ocftv'],
+    ],
+  }
+
   # Make pulse listen on localhost so desktops can SSH tunnel to it
   $tcp_module = 'load-module module-native-protocol-tcp listen=127.0.0.1 port=4713 auth-anonymous=1'
   exec { 'enable-pulse-tcp-listen':
     command => "echo '${tcp_module}' >> /etc/pulse/default.pa",
     unless  => "grep -q '^${tcp_module}$' /etc/pulse/default.pa",
     require => Package['pulseaudio'],
   }
```

Comment:
Should these configuration changes refresh the pulseaudio service?

Reply:
probably, yes. Using `notify => Service['pulseaudio']`?

Reply:
@kpengboy does this seem reasonable to you?